src/Microsoft.PowerApps.TestEngine/PowerFx/PowerFxEngine.cs

```diff
@@ -154,5 +153,11 @@ public IPowerAppFunctions GetPowerAppFunctions()
         {
             return _powerAppFunctions;
         }
+
+        public async Task RunRequirementsCheckAsync()
+        {
+            await _powerAppFunctions.CheckAndHandleIfLegacyPlayerAsync();
+            await _powerAppFunctions.TestEngineReady();
```

Comment:
this function returns true (posted another question there), but we're not reading that to confirm true/false state. Isn't there a scenario where we'd have to halt execution in a false state?

Reply:
It will never return false, but throws exception in that case. 
Also please read the PR description on why we are not halting the execution for various scenarios.